server/venueless/api/views.py

```diff
@@ -115,6 +119,90 @@ def get(self, request, **kwargs):
             )
 
 
+class CreateWorldView(APIView):
+    authentication_classes = []  # disables authentication
+    permission_classes = []
+
+    @staticmethod
+    def post(request, *args, **kwargs) -> JsonResponse:
+        payload = CreateWorldView.get_payload_from_token(request)
+
+        # check if user has permission to create world
+        if payload.get("has_permission"):
+            secret = get_random_string(length=64)
+            config = {
+                "JWT_secrets": [
+                    {
+                        "issuer": "any",
+                        "audience": "venueless",
+                        "secret": secret,
+                    }
+                ]
+            }
+
+            title_default = {
```

Comment:
This impementation leave to many holes:
- `data.get()` can return `None` and `None.items()` will throw exception.
- `set.pop()` will throw exception if `set` is empty.

Also, if you don't use `key`, retrieving values from dict instead.